samples/kotlin-android/build.gradle.kts

```diff
@@ -37,14 +37,14 @@ dependencies {
 }
 
 generateSwagger {
-    platform = "kotlin"
-    packageName = "com.yelp.codegen.samples"
-    specName = "sample_specs"
-    specVersion = "1.0.0"
-    inputFile = file("../sample_specs.json")
-    outputDir = file("./src/main/java/")
+    platform.set("kotlin")
+    packageName.set("com.yelp.codegen.samples")
+    specName.set("sample_specs")
+    specVersion.set("1.0.0")
+    inputFile.set(file("../sample_specs.json"))
+    outputDir.set(project.layout.buildDirectory.dir("./src/main/java/"))
```

Comment:
If you add the build directory, then please place the files inside `/gen` or `/generated`.

Reply:
I reverted it back to `projectDirectory.dir(".src/main/java"))`. 
It feels a bit weird to have generated files under version control though. Maybe we should move eveything under `/build` ?

Reply:
+1 to not placing generated files in VC. The whole point of swagger is to be schema driven, if you check in generated outputs, you run the risk of being out of sync with the schema, which sort of defeats the key purpose of this.

In this case if the inputs and outputs of the gradle tasks are setup correctly, the gradle build cache will avoid the work in a similar way that checking things in would, but without the risk of having your inputs and outputs getting out of sync.

Reply:
Agree, that's also the reason why `/src/main/java` is inside the `.gitignore`. I'm fine either way as long as we don't commit them.

Reply:
Some of the `junit-test` generated code is commited. See https://github.com/Yelp/swagger-gradle-codegen/tree/master/samples/junit-tests/src/main/java

Is that intentional ?